app/models/checkout.rb

```diff
@@ -10,4 +11,19 @@ class Checkout < ApplicationRecord
   def due_date
     self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
   end
+
+  def fine
+    calculate_overdue_penalty
```

Comment:
I think you could just as easily inline this here. The extraction doesn't seem to have much benefit. My original suggestion was to extract just the `days_overdue` method, but I don't feel strongly about that.